onnxruntime/contrib_ops/cpu/transformers/beam_search.cc

```diff
@@ -148,6 +150,37 @@ Status BeamSearch::SetupSubgraphExecutionInfo(const SessionState& session_state,
                                         t5_decoder_subgraph_->num_layers);
     }
   }
+  else if (parameters_.model_type == IGenerationParameters::kModelTypeWhisper) {
+    if (attribute_name == "encoder") {
+      ORT_ENFORCE(t5_encoder_subgraph_ == nullptr,
```

Comment:
>t5_encoder_subgraph_ [](http://example.com/codeflow?start=18&length=20)

Why do we need a specific type for Whisper? Aren't T5 and Whisper the same type?